node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,7 @@
+import { writeFile } from 'node:fs/promises';
+
+try {
+  await writeFile('random.txt', Math.random().toString() + '\n', 'utf');
```

Comment:
The `.toString` call is unnecessary since JS will convert it to a string anyway. It does that because of the `+ '\n'` -- when you concatenate something with a string, JS converts it to a string for you.